conf/server.yaml.template

```
@@ -50,6 +50,8 @@ SERVER:
   ADMIN_PASSWORD: changeme
 
   SSH_CLIENT:
+    # Specify port number for ssh client, Default: 22
+    SSH_PORT:
```

Please update this to match `settings.server.ssh_client.port`
```suggestion
    PORT:
```